server/command.go

```diff
@@ -75,10 +75,15 @@ func (p *Plugin) executeCommand(c *plugin.Context, args *model.CommandArgs) (str
 
 		meetingID := zoomUser.Pmi
 
-		_, appErr = p.postMeeting(user, meetingID, args.ChannelId, "")
+		var createdPost *model.Post
+		createdPost, appErr = p.postMeeting(user, meetingID, args.ChannelId, "")
 		if appErr != nil {
 			return "Failed to post message. Please try again.", nil
 		}
+
+		if appErr = p.API.KVSetWithExpiry(fmt.Sprintf("%v%v", postMeetingKey, meetingID), []byte(createdPost.Id), meetingPostIDTTL); appErr != nil {
+			p.API.LogDebug("failed to store post id", "err", appErr)
+		}
```

Comment:
Could this be a helper method to ensure the code is DRY?

Reply:
To avoid bugs like to one fixed with [https://github.com/mattermost/mattermost-plugin-zoom/pull/134/commits/6686411cff293068768bd02673c6dd75d2ce2782 ](https://github.com/mattermost/mattermost-plugin-zoom/pull/134/commits/6686411cff293068768bd02673c6dd75d2ce2782), maybe even posting the message should be part of the helper.